app/views/finance/financial_transactions/new_collection.html.haml

```diff
@@ -57,6 +57,10 @@
     = link_to t('.new_ordergroup'), '#', 'data-add-transaction' => true, class: 'btn'
     = link_to t('.add_all_ordergroups'), '#', 'data-add-all-ordergroups' => true, class: 'btn'
   - if BankAccount.any?
+    %p
+      %label
+        = check_box_tag :create_foodcoop_transaction, true, params[:create_foodcoop_transaction]
+        = t('.create_foodcoop_transaction')
```

Comment:
Why is this depending on BankAccount

Reply:
it's misused as feature flag ATM. same applies to the financial links in the lines below. i'd like to avoid additional settings as much as possible, since people already don't get the existing ones :disappointed:

Reply:
Ah :) I agree the configuration has become quite complex. Can we also keep the code clear somehow?
To my mind, I'd rather put this in configuration, and leave making that better for another job.

Is there a reason not to show the option for foodcoop-transactions by default? Could we make it not stand in the way?
If it does clutter the interface (here and perhaps at other places), than maybe a feature toggle would be good anyhow.

Reply:
That said, I think "support double entry accounting" could be a useful configuration option.

Reply:
i've change the check to existence of any foodcoop-related transaction for now. it's not perfect either, but it should do the job for now. i'd like to avoid addion of attional settings until everything related to finances is in `master`

Reply:
Ok, that makes sense. Would you mind creating an issue to keep track of that, so that these loose ends are somewhere public?

Reply:
added #741

Reply:
Thanks!